sugargame/canvas.py

```diff
@@ -1,21 +1,20 @@
 import os
 from gi.repository import Gtk
-from gi.repository import GObject
 from gi.repository import GLib
 from sugar3.activity.activity import PREVIEW_SIZE
 import pygame
-import event
+import sugargame.event as event
 
 CANVAS = None
 
 
 class PygameCanvas(Gtk.EventBox):
     def __init__(self, activity, pointer_hint=True,
```

Comment:
Remove `pointer_hint` from PygameCanvas, for reference see commit [59277d9](https://github.com/sugarlabs/sugargame/commit/59277d99d0bfabf4d9ef97ee7b99194723086df1#diff-0e7cb4c5bbdd52f79dd903d700945b92)

Reply:
Thanks I'll make changes.